docs/source/frameworks/pytorch.rst

```diff
@@ -2,55 +2,158 @@
 PyTorch
 =======
 
-Here's an example using PyTorch with BentoML:
+BentoML provides native support for serving and deploying models trained from PyTorch.
 
-.. code:: python
+Preface
+-------
+
+User may also convert PyTorch models to torchscript models, and use ``bentoml.torchscript`` 
+to save them. It will be a more stable way to serve and deploy PyTorch models in production.
+
+.. note::
+
+    :bdg-info:`Remarks:` We recommend users apply model optimization techniques such as **distillation** or **quantization**.
+    Alternatively, PyTorch models can also be converted to :ref:`ONNX <frameworks/onnx>` models and leverage different runtimes (e.g. TensorRT, Apache TVM, etc.).
```

Comment:
```suggestion
    :bdg-info:`Remarks:` We recommend users to apply model optimization techniques such as `distillation <https://arxiv.org/abs/1503.02531>`_ or `quantization <https://pytorch.org/docs/stable/quantization.html#general-quantization-flow>`_ .
    Alternatively, PyTorch models can also be converted to :doc:`/frameworks/onnx` models and leverage different runtimes (e.g. TensorRT, Apache TVM, etc.).
```

Reply:
add something like:
`in this tutorial, we will go over how to train a simple cnn with pytorch and use bentoml ...`